packages/composite-checkout/src/components/radio-button.js

```diff
@@ -124,13 +122,17 @@ export default function RadioButton( { checked, name, value, onChange, children
 				} }
 				readOnly={ ! onChange }
 			/>
-			<LabelContent checked={ checked }>{ children }</LabelContent>
-		</Label>
+			<Label checked={ checked } htmlFor={ value }>
```

Comment:
Since the composite checkout component should be able to be embedded inside any page, and since it can contain custom content as well, setting an `id` attribute for an html element is tricky because that attribute should be unique on the page.

If we wrap the radio button inside the `<label>` tag, we don't need the `id` prop (or the `htmlFor` prop).

Reply:
> If we wrap the radio button inside the <label> tag, we don't need the id prop (or the htmlFor prop).

That's true the problem is it limits what we can add as a child for this component. Placing block level elements, like the `divs` for the credit card field, is invalid HTML markup. We could explicitly pass a required id prop to the component to prevent any duplicates from appearing on the same page. What do you think?